static/downloadables/runPrsCLI.sh

```diff
@@ -239,16 +239,25 @@ learnAboutParameters () {
                 echo -e "in the input VCF file. Available options are ${GREEN}hg17${NC}, ${GREEN}hg18${NC}, ${GREEN}hg19${NC}, and ${GREEN}hg38${NC}."
                 echo "" ;;
             5 ) echo -e "${MYSTERYCOLOR}-p Preferred GWA Study Super Population: ${NC}"
-	    #TODO: write a better explanation for this section
                 echo "This parameter is required for us to run Linkage Disequilibrium on "
                 echo "SNPs for PRS calculation. We use the five super populations from the " 
-                echo "1000 Genomes as the available options. Below are the acceptable codes. " # this will need some re-work on the language
-                echo "" #AFR, AMR, EAS, EUR, SAS
+                echo "1000 Genomes as the available options. Most studies have one or more super"
+                echo "populations reported. We ask users for their preferred super population to"
+                echo "help us choose which super population to use if there is more than one reported"
+                echo "or if no super populations are reported. Below are the acceptable codes "
+                echo "and the order super populations are chosen based on preferred super population."
```

Comment:
The last line here needs some clarification. If I choose "AFR," shouldn't I expect to get "AFR"? Why would PRSKB choose a different super pop from the lists below?